index.html

```diff
@@ -6,6 +6,15 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
+    <h>Hallo</h>
+    <nav>
+        <ul>
+            <li>home</li>
+            <li>search</li>
+            <li>over</li>
+            <li>zoekpagina</li>
+        </ul>
```

Comment:
om toch nog wat toe te kunnen voegen straks, maak ook maar een login in de lijst.